Build/CMake/Modules/AtomicCommon.cmake

```diff
@@ -1,5 +1,5 @@
 #
-# Portions Copyright (c) 2008-2016 the Urho3D project.
+# Portions Copyright (c) 2008-2017 the Atomic project.
 #
```

Comment:
Let's keep the Urho3D project copyright intact, some of the cmake script here is for compatibility, especially with ThirdParty